cypress/integration/Tooltip.spec.ts

```diff
@@ -0,0 +1,126 @@
+import * as h from '../helpers';
+
+describe('Tooltip', () => {
+  before(() => {
+    h.stories.visit();
+  });
+
+  context('given Default is rendered', () => {
+    beforeEach(() => {
+      h.stories.load('Components|Popups/Tooltip/React', 'Default');
+    });
+
+    it('should not have any axe errors', () => {
+      cy.checkA11y();
+    });
+
+    it('should have an aria-label of "Close"', () => {
+      cy.get('button').should('have.ariaLabel', 'Close');
+    });
+
+    context('when close icon is hovered', () => {
+      beforeEach(() => {
+        cy.get('button').trigger('mouseover');
+      });
+
+      it('should open the tooltip', () => {
+        cy.findByRole('tooltip').should('be.visible');
+      });
+
+      it('should not have any axe errors', () => {
+        cy.checkA11y();
+      });
+
+      context('when the tooltip is hovered', () => {
+        beforeEach(() => {
+          cy.get('button').trigger('mouseout');
+          cy.findByRole('tooltip').trigger('mouseover');
+        });
+
+        it('should not close the tooltip', () => {
+          cy.findByRole('tooltip').should('be.visible');
+        });
+      });
+
+      context('when ESC key is pressed', () => {
+        beforeEach(() => {
+          cy.get('body').trigger('keydown', {
+            key: 'Escape',
+          });
+        });
+
+        // TODO: implement this functionality
+        it.skip('should close the tooltip', () => {
+          cy.findByRole('tooltip').should('not.be.visible');
+        });
```

Comment:
There's some WIP on the accessibility specifications and some of them we're punting on until we have a Popup manager or even more time (like collision detection)